### PR TITLE
Refactor layouts

### PR DESCRIPTION
**Pull request by Itay4**
##  Status
- [x] Ready

### CODE REVIEW

File: Packs/Cymulate/Layouts/layout-details-Cymulate_Immediate_Threats-V2.json [hunk 1]
```diff
@@ -344,5 +344,6 @@
         "version":-1
     },
     "typeId":"Cymulate Immediate Threats",
-    "version":-1
+    "version":-1,
+    "fromVersion": "5.9.9"
```
Comment (aviadl) [inline Packs/Cymulate/Layouts/layout-details-Cymulate_Immediate_Threats-V2.json:348]: toVersion

File: Packs/DigitalGuardian/Layouts/layout-details-Digital_Guardian_Security_Event.json [hunk 2]
```diff
@@ -553,6 +553,7 @@
             }
         ],
         "typeId": "Digital Guardian Security Event",
-        "version": -1
+        "version": -1,
+        "fromVersion": "5.9.9"
```
Comment (aviadl) [inline Packs/DigitalGuardian/Layouts/layout-details-Digital_Guardian_Security_Event.json:557]: toVersion

File: Packs/EmployeeOffboarding/Layouts/layout-details-Employee_Offboarding-V2.json [hunk 3]
```diff
@@ -683,5 +683,6 @@
 		"version": -1
 	},
 	"typeId": "Employee Offboarding",
-	"version": -1
+	"version": -1,
+	"fromVersion": "5.9.9"
```
Comment (aviadl) [inline Packs/EmployeeOffboarding/Layouts/layout-details-Employee_Offboarding-V2.json:687]: toVersion

File: Packs/BPA/Layouts/layoutscontainer-Panorama_Best_Practice_Assessment.json [hunk 4]
```diff
@@ -0,0 +1,297 @@
+{
+    "id": "IllusionBLACK",
+    "group": "incident",
+    "name": "IllusionBLACK Incident Layout",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "details": {
```
Comment (aviadl) [inline Packs/BPA/Layouts/layoutscontainer-Panorama_Best_Practice_Assessment.json:7]: detailsV2

File: Packs/CommonTypes/Layouts/layoutscontainer-cve.json [hunk 5]
```diff
@@ -0,0 +1,327 @@
+{
+	"id": "cveRep",
+	"group": "indicator",
+	"name": "CVE Indicator Layout",
+	"version": -1,
+	"fromVersion": "6.0.0",
+    "indicatorsDetails": {
```
Comment (aviadl) [inline Packs/CommonTypes/Layouts/layoutscontainer-cve.json:7]: align

File: Packs/CommonTypes/Layouts/layoutscontainer-ip.json [hunk 6]
```diff
@@ -0,0 +1,425 @@
+{
+	"id": "ipRep",
+	"group": "indicator",
+	"name": "IP Indicator Layout",
+	"version": -1,
+	"fromVersion": "6.0.0",
+    "indicatorsDetails": {
```
Comment (aviadl) [inline Packs/CommonTypes/Layouts/layoutscontainer-ip.json:7]: align

**Comment by anara123** [comment]
@Itay4 the layout files downloaded from the server with `layoutcontainer-` prefix?

File: Packs/CommonTypes/Layouts/layoutscontainer-Access.json [hunk 7]
```diff
@@ -1,7 +1,8 @@
 {
     "id": "Access",
     "group": "incident",
-    "name": "Access Incident Layout",
+    "name": "Access Incident",
+	"description": "",
```
Comment (aviadl) [inline Packs/CommonTypes/Layouts/layoutscontainer-Access.json:5]: lint

File: Packs/CommonTypes/Layouts/layoutscontainer-Account.json [hunk 8]
```diff
@@ -1,7 +1,8 @@
 {
     "id": "accountRep",
     "group": "indicator",
-    "name": "Account Indicator Layout",
+    "name": "Account Indicator",
+	"description": "",
```
Comment (aviadl) [inline Packs/CommonTypes/Layouts/layoutscontainer-Account.json:5]: lint

File: Packs/CommonTypes/Layouts/layoutscontainer-Host.json [hunk 9]
```diff
@@ -1,7 +1,8 @@
 {
     "id": "hostRep",
     "group": "indicator",
-    "name": "Host Indicator Layout",
+    "name": "Host Indicator",
+	"description": "",
```
Comment (aviadl) [inline Packs/CommonTypes/Layouts/layoutscontainer-Host.json:5]: lint

**Comment by Itay4** [comment]
> Some linting comments
> But you didnt fix all other comments ...

might of missed, but didnt see other comments that were not addressed - can you point at them please?

File: Packs/CommonTypes/Layouts/layout-edit-Access.json [hunk 10]
```diff
@@ -1,186 +0,0 @@
-{
```
Comment (aviadl) [inline Packs/CommonTypes/Layouts/layout-edit-Access.json:1]: Why did you remove this file, and not add a `toversion` ?

File: Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json [hunk 11]
```diff
@@ -0,0 +1,423 @@
+{
+	"id": "Vulnerability",
+	"group": "incident",
+	"name": "Vulnerability Incident",
+	"description": "",
+	"version": -1,
+	"fromVersion": "6.0.0",
+    "edit": {
```
Comment (aviadl) [inline Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json:8]: Doesnt this have details/detailsv2 ?

File: Packs/Cymulate/Layouts/layoutscontainer-Cymulate_Immediate_Threats.json [hunk 12]
```diff
@@ -0,0 +1,341 @@
+{
+    "id": "Cymulate Immediate Threats",
+    "group": "incident",
+    "name": "Cymulate Immediate Threat",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
```
Comment (aviadl) [inline Packs/Cymulate/Layouts/layoutscontainer-Cymulate_Immediate_Threats.json:10]: Not critical, but since this is from 6.0 only
I would remove the legacy summary

File: Packs/EmployeeOffboarding/Layouts/layoutscontainer-Employee_Offboarding.json [hunk 13]
```diff
@@ -0,0 +1,810 @@
...
+    "id": "Employee Offboarding",
+    "group": "incident",
+    "name": "Employee Offboarding Incident",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
```
Comment (aviadl) [inline Packs/EmployeeOffboarding/Layouts/layoutscontainer-Employee_Offboarding.json:11]: Same here

File: Packs/Expanse/Layouts/layoutscontainer-Expanse_Appearance.json [hunk 14]
```diff
@@ -0,0 +1,315 @@
...
+    "id": "Expanse Appearance",
+    "group": "incident",
+    "name": "Expanse Appearance Incident",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
```
Comment (aviadl) [inline Packs/Expanse/Layouts/layoutscontainer-Expanse_Appearance.json:11]: Same here

File: Packs/Logzio/Layouts/layoutscontainer-Logz.io_Alert.json [hunk 15]
```diff
@@ -0,0 +1,377 @@
...
+    "id": "Logz.io Alert",
+    "group": "incident",
+    "name": "Logz.io Alert",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
```
Comment (aviadl) [inline Packs/Logzio/Layouts/layoutscontainer-Logz.io_Alert.json:11]: Same here

File: Packs/PcapAnalysis/Layouts/layoutscontainer-PCAP_Analysis.json [hunk 16]
```diff
@@ -0,0 +1,493 @@
...
+    "id": "PCAP Analysis",
+    "group": "incident",
+    "name": "PCAP Analysis Incident",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
```
Comment (aviadl) [inline Packs/PcapAnalysis/Layouts/layoutscontainer-PCAP_Analysis.json:11]: and here

File: Packs/Pcysys/Layouts/layoutscontainer-Pentera_Insight.json [hunk 17]
```diff
@@ -0,0 +1,329 @@
...
+    "id": "Pentera Insight",
+    "group": "incident",
+    "name": "Pentera Insight",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
```
Comment (aviadl) [inline Packs/Pcysys/Layouts/layoutscontainer-Pentera_Insight.json:11]: and here

File: Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit.json [hunk 18]
```diff
@@ -0,0 +1,385 @@
...
+    "id": "Prisma Cloud Compute Audit",
+    "group": "incident",
+    "name": "Prisma Cloud Compute Audit",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
```
Comment (aviadl) [inline Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit.json:11]: same here

File: Packs/Smokescreen_IllusionBLACK/Layouts/layoutscontainer-IllusionBLACK.json [hunk 19]
```diff
@@ -0,0 +1,375 @@
...
+    "id": "IllusionBLACK",
+    "group": "incident",
+    "name": "IllusionBLACK Incident",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
```
Comment (aviadl) [inline Packs/Smokescreen_IllusionBLACK/Layouts/layoutscontainer-IllusionBLACK.json:11]: Same here

File: Tests/Marketplace/marketplace_services.py [hunk 20]
```diff
@@ -881,7 +881,7 @@ def collect_content_items(self):
                 PackFolders.INDICATOR_FIELDS.value: "indicatorfield",
                 PackFolders.REPORTS.value: "report",
                 PackFolders.INDICATOR_TYPES.value: "reputation",
-                PackFolders.LAYOUTS.value: "layout",
+                PackFolders.LAYOUTS.value: "layoutscontainer",
```
Comment (aviadl) [inline Tests/Marketplace/marketplace_services.py:884]: The folder name here is not alligned with the file structure in the PR

File: Packs/CommonTypes/Layouts/layout-edit-Access.json (hunk 10 above)
Comment (Itay4) [reply Packs/CommonTypes/Layouts/layout-edit-Access.json:1]: didnt really remove it, just moved it to `AccessInvestigation` content pack and added `toVersion`
not sure why github shows it like that

File: Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json (hunk 11 above)
Comment (Itay4) [reply Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json:8]: it does have `details`, adding it

**Comment by Itay4** [comment]
build fails on non relevant validations and tests
merging
